include/drivers/can.h

```diff
@@ -47,6 +47,9 @@ extern "C" {
 /** unexpected error */
 #define CAN_TX_UNKNOWN  (-5)
 
+/** invalid parameter */
+#define CAN_TX_EINVAL   (-22)
+
```

Comment:
Why not just use -EINVAL?

Reply:
Because we don't use POSIX error codes in CAN, and this way, we can guarantee that the EINVAL doesn't overlap with some other CAN error code by accident. 22 is actually EINVAL

Reply:
OK.